{
	"next_topic_id": 1,
	"topic": [],
	"topic_vote": {},
	"next_comment_id": 2,
	"comment": {
		"1539142010_1GVM7nNzmYJ3ppPCmDTPLozrJF2Z9mgFpF": [
			{
				"comment_id": 1,
				"body": "Errors?!\nRyzen 7 3700X\n\nBenchmarking ZeroNet 0.7.1 (rev4260) Python 3.8.0 (default, Oct 23 2019, 18:51:26) [GCC 9.2.0] on: linux...\n\nCryptBitcoin:\n- hdPrivatekey x 10..........0.089s [x7.89: Insane!!]\n- sign x 10..........0.093s [x3.76: Insane!!]\n- btctools verify x 100..........0.672s [x0.55: Goodish]\n- Error loading openssl: No module named 'bitcoin'\n- Error loading libsecp256k1: No module named 'coincurve'\n\nCryptHash:\n- sha256 5M x 10..........0.027s [x22.60: Insane!!]\n- sha512 5M x 10..........0.061s [x9.83: Insane!!]\n- os.urandom(256) x 1000..........0.006s [x1.09: OK]\n\nMsgpack: (version: 0.6.2)\n- pack 5K x 10 000..........0.014s [x54.42: Insane!!]\n- unpack 5K x 10 000..........0.022s [x55.11: Insane!!]\n- streaming unpack 5K x 10 000 (fallback: True)..........0.196s [x7.16: Insane!!]\n- streaming unpack 5K x 10 000 (fallback: False)..........0.014s [x100.00: Insane!!]\n\nDb: (version: 3.30.1, API: 2.6.0)\n- Open x 10..........0.012s [x10.43: Insane!!]\n- Insert x 10 x 1000..........0.114s [x8.76: Insane!!]\n- Buffered insert x 100 x 100..........0.185s [x7.02: Insane!!]\n+ Total rows in db: 20000\n- Indexed query x 1000..........0.029s [x8.59: Insane!!]\n- Not indexed query x 100..........0.090s [x6.68: Insane!!]\n- Like query x 100..........0.179s [x10.05: Insane!!]\n\nCompression:\n- Zip pack x 10..........0.042s [x2.84: WOW]\n- Zip unpack x 10..........0.042s [x4.71: Insane!!]\n- Tar.gz pack x 10..........\n! Error: AssertionError: Invalid hash: 9715c057620e8538e0c38edeac737a825ef1c24ba19bd190adf83499e9c19f0b != 4704ebd8c987ed6f833059f1de9c475d443b0539b8d4c4cb8b49b26f7bbf2d19\nin Stats/StatsPlugin.py line 500 > Stats/StatsPlugin.py line 822\n0.114s [x2.64: WOW]\n- Tar.gz unpack (2.53kB) x 10..........0.103s [x1.94: Fast]\n- Tar.bz2 pack x 10..........\n! Error: AssertionError: Invalid hash: ad13921c9eecf309743e89e645b3594a83eea4a8621c2fca49bb9dadf8c2de67 != 90cba0b4d9abaa37b830bf37e4adba93bfd183e095b489ebee62aaa94339f3b5\nin Stats/StatsPlugin.py line 500 > Stats/StatsPlugin.py line 822\n0.701s [x2.85: WOW]\n- Tar.bz2 unpack (0.82kB) x 10..........0.206s [x2.43: Fast]\n- Tar.xz pack x 10..........\n! Error: AssertionError: Invalid hash: 70e20073861bacc920fb89add7362b37a4a2c27ac20b2a1edea990af76e649ae != 37abc16d552cfd4a495cb2acbf8b1d5877631d084f6571f4d6544bc548c69bae\nin Stats/StatsPlugin.py line 500 > Stats/StatsPlugin.py line 822\n0.254s [x5.52: Insane!!]\n- Tar.xz unpack (1.02kB) x 10..........0.093s [x2.15: Fast]\n\nDone. Total: 3.65s",
				"added": 1573894372
			}
		]
	},
	"comment_vote": {}
}